Daytime-Nighttime-Interactions/fault-handling-workflow.rst

```diff
@@ -0,0 +1,164 @@
+
+.. note::
+
+  Throughout this section the role of *Triage Manager* is referenced.
+  This position is a rotating role that is responsible for managing the assignment of tickets created during the night, as well as following up on the urgent tickets during the day.
+
+
+OBS Jira Project Description
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The workflow as executed in Jira passes through several states, beginning with the reported state, where it receives and assignee, through to one of the final completed states.
+
+The creation of a ticket can be performed by anyone, but is generally done by whomever is operating the system.
+Details on ticket creation on the fields are found in the page on :ref:`Daytime-Nighttime-Interactions-fault-reporting`.
+
+The Jira workflow for the OBS project is shown in the following figure and the various states and transitions are detailed below.
+
+.. figure:: _static/OBS_Jira_workflow.png
+    :name: OBJ Jira Project Workflow
+    :width: 621
+    :height: 422
+
+    A block diagram of the workflow contained in the OBS Jira project.
+    Note that the transition labels in this diagram describe the process and do not exactly match the names used by the OBS Jira project itself.
+
+.. :target: ../../../Daytime-Nighttime-Interactions/_static/OBS_Jira_workflow.png
+..     :target: _static/OBS_Jira_workflow.png
+
+Final states
+------------
+The final states are the ultimate destination of a reported ticket.
+When a ticket reaches this state, is it considered to be resolved and will no longer be actively managed by the Fault Handling Management process.
+We start by explaining these states because they are referenced throughout the remainder of the document, specifically when describing state transitions.
+
+- ``Done``: All work required to address the issue has been completed. 
+
+- ``Cancelled``:  The reported issue is no longer valid, was filed by mistake, or has been superseded by another ticket or other events
+
+- ``Transferred``: The issue has been deemed non-critical to immediate operations and has been moved to a subsystem for consideration and/or implementation.
+
+- ``Duplicate``: This is a repeat reporting of an already in-process ticket. 
+  Work on the issue should proceed in the original ticket.
+
+There are multiple ways that tickets reach these states, which are explained in the sections below.
+The ``Cancelled`` and ``Duplicate`` states are special because they can be transitioned to from any active state in the workflow.
+
+There is one transition out of these states, which is from the ``Done`` state to the ``Reported`` state.
+This transition should be rare. 
+It is reserved for cases when a job is in a completed state, but either the exact error occurs again, or someone deems it is not satisfactorily completed. 
+The ticket is then transitioned to ``Reported`` so it can be triaged and prioritized accordingly by a Triage manager.
+
+
+Active states
```

Comment:
I actually did this originally and then removed it.
I have mixed feelings about it because in some ways I don't know that they are really sections as then they go into tables of contents etc.
I think the nail in the coffin was actually linked to the highlighting issue, if you change it to a section, then you can link the section, which is good, but you can't do the text highlighting, which is bad.

Reply:
That's a bummer, okay we can stick with the formatting as-is for now.